npm run tsc gives me
src/hooks.ts:84:5 - error TS2552: Cannot find name 'countRows'. Did you mean 'useCountRows'?

I believe you're missing an import ...

--- a/src/hooks.ts
+++ b/src/hooks.ts
@@ -1,5 +1,5 @@
 import { useState, useEffect } from "react";
-import { fetchOneRow, fetchRows } from "./api.js";
+import { fetchOneRow, fetchRows, countRows } from "./api.js";
 
 interface HookFetchResult<T> {
   rows: T[];

